### PR TITLE
Added tooltips to the Performance Example for WPF

### PR DESCRIPTION
**Pull request by GeertvanHorrik**
Still working on this PR, but this is the result of #219

### CODE REVIEW

**Comment by GeertvanHorrik** [comment]
See the image, the problem is that the tooltips are visible on the location where you enter the control (which looks really weird):

image

**Comment by GeertvanHorrik** [comment]
When using ToolTipService.Placement="center", the tooltip is correctly aligned. I think using a custom callback is the solution.

**Comment by GeertvanHorrik** [comment]
The readibility constraints are a bit too strict imho, I must force this:

```
            ToolTipService.AddToolTipOpeningHandler(this, (sender, args) =>
            {
                var toolTip = args.Source as ToolTip;
                if (toolTip != null)
                {
                    var mousePosition = Mouse.GetPosition(this);

                    toolTip.Placement = PlacementMode.RelativePoint;
                    toolTip.PlacementTarget = this;
                    toolTip.HorizontalOffset = mousePosition.X;
                    toolTip.VerticalOffset = mousePosition.Y;
                }
            });
```

into this:

```
            ToolTipService.AddToolTipOpeningHandler(
                this,
                (sender, args) =>
            {
                var toolTip = args.Source as ToolTip;
                if (toolTip != null)
                {
                    var mousePosition = Mouse.GetPosition(this);

                    toolTip.Placement = PlacementMode.RelativePoint;
                    toolTip.PlacementTarget = this;
                    toolTip.HorizontalOffset = mousePosition.X;
                    toolTip.VerticalOffset = mousePosition.Y;
                }
            });
```

**Comment by objorke** [comment]
Why did you add this to the performance example? Wouldn't it be better to create a new example just demonstrating the tooltip capability?

**Comment by GeertvanHorrik** [comment]
To check if I can reproduce. Yes, it might be better, but as with everyone, my time is limited but trying to do the right thing. Let me first finish the issue, then we can check if we need a separate example.

**Comment by GeertvanHorrik** [comment]
@objorke Just added TitleToolTip. I need some help with rendering though. I see in other places (annotations, series, etc) that you set tooltip, render, then clear it again. However, I don't see how they are actually rendered / set to an element.

**Comment by GeertvanHorrik** [comment]
@objorke never mind, found it. I need to set the tooltip before rendering, then it gets applied automatically.

**Comment by GeertvanHorrik** [comment]
Also added the demo. Please review this PR.

**Comment by GeertvanHorrik** [comment]
"Write-Error : Package 'gtksharp v2.12.26' did not install successfully: [ERROR]
 Running msiexec with /i "C:\Users\appveyor\AppData\Local\Temp\1\chocolatey\gtk
sharp\gtksharpInstall.msi" /quiet /qn /norestart  was not successful. Exit code
 was '1603'."

Is that something I caused?

**Comment by GeertvanHorrik** [comment]
No, it seems this is for all users: #516

**Comment by GeertvanHorrik** [comment]
Any updates on this?

File: Source/OxyPlot.Wpf/Plot.cs [hunk 1]
```diff
@@ -153,6 +153,24 @@ private static void AppearanceChanged(DependencyObject d, DependencyPropertyChan
...
+        /// <param name="d">The d.</param>
+        /// <param name="e">The <see cref="System.Windows.DependencyPropertyChangedEventArgs" /> instance containing the event data.</param>
+        private static void TitleToolTipChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var plot = (Plot)d;
+
+            var model = plot.ActualModel;
+            if (model != null)
+            {
+                model.TitleToolTip = e.NewValue as string;
```
Comment (tibel) [inline Source/OxyPlot.Wpf/Plot.cs]: For other properties this is handled in `OnAppearanceChanged`. Why the different handling?

Comment (GeertvanHorrik) [inline Source/OxyPlot.Wpf/Plot.cs]: I overlooked the SynchronizeProperties. Will update.

**Comment by GeertvanHorrik** [comment]
Done.

**Comment by tibel** [comment]
And update CHANGELOG.md, please.

**Comment by GeertvanHorrik** [comment]
Changelog is updated.

**Pull request merged**